nss-day-cohort-48/djv_truncheons-and-flagons
# Moved persistent data to a json-server API

## To Test
- pull down the code (and make sure you're in the repo's root directory)
- `serve` and `json-server --port 8080 api/db.json`
- if you need to reset the database, run `dev-scripts/init-database.sh` **and** close/reopen the json-server

## Look For
- you should be able to enter scores and submit them
- check to see that the current game scores at the bottom update accordingly
- check to see that the leaderboard at the top updates accordingly

## Known Issues
- the round number just keeps going up and up and up -- needs to cap at 3
